firebolt_provider/hooks/firebolt.py

```diff
@@ -51,21 +73,20 @@ class FireboltHook(DbApiHook):
     def get_ui_field_behaviour() -> Dict:
         """Returns custom field behaviour"""
         return {
-            "hidden_fields": ["port"],
+            "hidden_fields": ["port", "host"],
             "relabeling": {
                 "schema": "Database",
                 "login": "Username",
                 "extra": "Advanced Connection Properties",
             },
             "placeholders": {
-                "host": "The host name of your Firebolt database",
```

Comment:
Does this mean we can't run airflow against dev anymore? If so, I'd be against such change. Especially since our other connectors expose this field.

Reply:
In this case I will add a hidden `api_endpoint` parameter for the extra field in configuration. This parameter is needed for the internal development and causes some confusion for client

Reply:
Yeah, if it's possible to add a hidden api_endpoint param it would be great. Since it won't be in public docs, can you add a quick line about it in Guru so we have somewhere to refer to later on?

Reply:
https://app.getguru.com/card/idAd4q4T/Set-custom-api_endpoint-for-airflow-provider